tests/simulation/test_simulator.py

```diff
@@ -27,6 +27,7 @@ def test_one_node(self):
         # FIXME: the setup above produces 0 new blocks and transactions
         # self.assertGreater(manager1.tx_storage.get_vertices_count(), 3)
 
+    @pytest.mark.flaky(max_runs=5, min_passes=1)
```

Comment:
I feel that sync-v2 would not affect how flaky this test is, so we could remove this marker without prejudice.

Reply:
I removed this from the base test and moved it to the sync-v2 base so it doesn't affect how the test runs on sync-v1, but I still left the flaky mark on the sync-v2 tests.